flash/core/data/utils.py

```diff
@@ -162,7 +164,7 @@ def image_default_loader(file_path: str, drop_alpha: bool = True) -> Image:
         file_path: The image file to load.
         drop_alpha: If ``True`` (default) then any alpha channels will be silently removed.
     """
-    img = default_loader(file_path)
+    img = default_loader(file_path) if _TORCHVISION_AVAILABLE else pil_image.open(file_path)
```

Comment:
From my suggestion above:

```suggestion
    img = default_loader(file_path) if _TORCHVISION_AVAILABLE else PILImage.open(file_path)
```